articles/what-is-anti-virus-software/index.md

```diff
@@ -0,0 +1,179 @@
+
+Call In The Calvary
+-------------------
+
+In addition to common good internet practices the second best thing to invest in is anti-malware/antivirus software. Anti-malware’s main objective is to stop any malicious threats from infecting your computer; those include a platitude of threats ranging from computer viruses to internet worms.
+
+Not having AV technology can lead to malware breeches. Currently, malware breeches of all sizes are costing industries $350 billion a year! That is why cyber-breeches incentivize companies and governments to heavily invest in cyber-security.
+
+Anti-malware software is thus an incredible useful tool to help identify and eliminate many malware-threats.
+
+_In this article I will go over the reason why anti-malware is important and what types of malware are out there. Furthermore, I will go in depth over the certain techniques used to do malware analysis and investigations. Lastly, there will a reading list to give you an review of some great anti-malware software out there.
+
+Team Red: Malicious Rogue Bandits
+=================================
+
+![Bandits and Roguges](/articles/what-is-anti-virus-software/rogue-bandits.jpg)
+
+What bandits and gangs were to the Wild West are hackers and scammers in the 21st century. Only difference — they use malware instead of guns to attack and exfiltrate personal data.
+
+Overview: What exactly is Malware?
+----------------------------------
+
+According to Malware bytes Labs, [Malware](https://blog.malwarebytes.com/glossary/malware/) or malicious software “is an umbrella term that describes any malicious program or code that is harmful to systems.” Malware is usually intentionally designed to either destroy data, install additional programs or exfiltrate (steal) data. For instance, malware can break computer systems or even be part of a larger agenda in the case of Wanna Cry. In summary, malware compromises the confidentiality, integrity and availability of a victim’s data (CIA for short).
+
+It is important to understand why malware is a huge concern. Companies large and small are constant targets to malware attacks.
+
+Furthermore, individuals who aren’t likely end-targets can still become potentially impacted as an intermediary; that is, they can be used, for example, as bots in much larger botnet to do more major attacks like dDoS(distributed denial of service) attacks etc. These attacks typically result in loss of data, intellectual property, competitive advantage, and loss of overall consumer confidence.
+
+Malware Arsenal
+===============
+
+![bandit_arsenal](/articles/what-is-anti-virus-software/bandit_arsenal.jpg)
+
+Let’s now look at some of the major forms of malware.
+
+Viruses
+-------
+
+Computer viruses can run on any operating system be it Windows or Mac ( Mac’s are not immune ;) ). Computer viruses are actually, contrary to common belief, a type of malware. Viruses are defined as follows:
```

Comment:
If you like you can reference this recent article which found Mac has become more susceptible than Windows:
https://blog.malwarebytes.com/reports/2020/02/malwarebytes-labs-releases-2020-state-of-malware-report/amp/

Reply:
I added the link to the Malwarebytes
Thanks

Reply:
added link to malwarebytes